src/main/java/org/seattlevoluntech/controllers/UserController.java

```diff
@@ -1,40 +1,42 @@
 package org.seattlevoluntech.controllers;
 
 import com.google.common.collect.Lists;
+import org.seattlevoluntech.storage.Project;
 import org.seattlevoluntech.storage.User;
 import org.seattlevoluntech.storage.UsersRepository;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.*;
 
 import javax.servlet.http.HttpServletRequest;
 import java.util.List;
+import java.util.Optional;
 
 @RestController
 public class UserController {
-
-    @Autowired UsersRepository userRepository;
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-    @RequestMapping("/users")
+    @Autowired
+    UsersRepository userRepository;
+
+    @GetMapping("/user")
```

Comment:
should be "/users"

Reply:
Looks accurate, I think this is pretty much what Tara and I were working towards but at a much slower pace haha. You guys are so fast!

Reply:
done